### ISSUE TITLE
Problem in finetune in other languages

### ISSUE THREAD

**Issue by Ashish-Patnaik** [k2-fsa/ZipVoice#118]
i want to finetune this model on google colab on other language and i finetuned also but the results are very poor it still can't speak the test ,so can someone give me a colab script to finetune this model on another language   and also tell how many epoch should i wait till the model decently speak the text with a reference audio accent

**Comment by zhu-han** [comment]
How much data is in your training set? 1000 epoch is a big number. Fine-tuning usually doesn’t require that many epochs.

**Comment by Ashish-Patnaik** [comment]
My dataset is 1000 samples of Hindi audio around 1 hour and every epoch is just taking 4-5 seconds in p100 gpu in kaggle/colab

**Comment by Ashish-Patnaik** [comment]
Thanks for your reply , I will apply the changes you said and go for 16 hours of data to fine-tune and increase the learning rate ,then I will update you

**Comment by Ashish-Patnaik** [comment]
Just one query can I increase the learning rate more than the default value so that the model learns the language better

**Comment by zhu-han** [comment]
> Just one query can I increase the learning rate more than the default value so that the model learns the language better

Yes, you can.

**Comment by lalimili6** [comment]
Dear @Ashish-Patnaik 
Common vice don't have speaker label, how can use this dataset for training?
Can we use datasets without speaker labels for training?
Best regards

**Comment by utkarshshukla2912** [comment]
@Ashish-Patnaik did you get some good results, what hyperparameters worked for you ?

**Comment by Ashish-Patnaik** [comment]
@utkarshshukla2912  No, not getting any good results. It just produce noise
